doc/plugin_server_nodeattestor_aws_iid.md

```diff
@@ -25,6 +26,12 @@ A sample configuration:
         }
     }
 ```
+
+## Disabling Instance Profile Selectors
+In cases where spire-server is running in a location with no public internet access available, setting `disable_instance_profile_selectors = true` will prevent the server from making requests to `iam.amazonaws.com`. This is needed as spire-server will fail to start.
```

Comment:
> This is needed as spire-server will fail to start.

I could see it failing agent attestation, but how does it cause the server to not start?

Reply:
Ah yes, you're right. Let me fix the wording of that.